app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  def new
+    #redirect_to dashboard_path if logged_in?
+  end
+
+  def create
+    auth = env["omniauth.auth"]
+    require "pry"; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.